src/components/ShareButton/ShareButton.st.css

```diff
@@ -3,13 +3,16 @@
     -st-named: color, fallback
 }
 
+/*Defaults*/
 :vars {
-    /*Defaults*/
     DefaultColor: color-5;
 }
 
+/*Overrides*/
 :vars {
-    /*Overrides*/
+    /*
+    @default color-5
+    */
     IconColor: --overridable;
```

Comment:
What about the description for the var?
I know it is not really needed and the meaning is clear from the variable name but since we have the description column it might be empty in this case.

/*
        Icon Color
        @default color-1
*/

Reply:
If it's exactly the same as the variable name it sounds very redundant to me...
I'm not sure it's worth the effort.

WDYT?